# This service allows the user to change the parameters objRec uses for 
#  matching point clouds

float64 normal_radius
float64 feature_radius
int64 num_samples
float64 min_sample_dist
int64 k_correspondences

---

int64 ret           # Whether we were successfully able to change bounds
string msg          # Any error message associated with it

